### ISSUE TITLE
torrenttunes.tk is down

### ISSUE THREAD

**Issue by lextiz** [dessalines/torrenttunes-client#101]
The link from the readme.md (torrenttunes.tk) points to site that does not work. It returns 502.

**Comment by dessalines** [comment]
That's correct, this did rely on a backend for its search engine, which is down. Unless the project gets more interest, I probably won't restart it back up again as it takes a lot of processing power to seed thousands of torrents.

**Comment by lextiz** [comment]
I understand your concerns, but this looks like a loop: how the project could possibly get more attention if it is not usable?
